src/test/java/org/kohsuke/github/GHOrganizationTest.java

```diff
@@ -76,6 +77,15 @@ public void testInviteUser() throws IOException {
         // assertTrue(org.hasMember(user));
     }
 
+    @Test
+    public void testListMembersWithRole() throws IOException {
+        GHOrganization org = gitHub.getOrganization(GITHUB_API_TEST_ORG);
+
+        List<GHUser> admins = org.listMembersWithRole("admin").asList();
+
+        assertFalse(admins.isEmpty());
```

Comment:
Go ahead and be specific about size and who is in the list.  A non-empty list is going to be enough to ensure this doesn't break.